tests/schema/types/test_date.py

```diff
@@ -105,7 +106,8 @@ def test_serialization_of_incorrect_date_string(value):
     result = execute_mutation(value)
     assert result.errors
     assert isinstance(result.errors[0], GraphQLError)
-    assert result.errors[0].original_error is None
+    if IS_GQL_32:
+        assert result.errors[0].original_error is None
```

Comment:
how's this in 3.3?

Reply:
![image](https://github.com/user-attachments/assets/0d96483d-8df7-44bd-975b-11c186418cee)

Reply:
Yeah I have no clue. :disappointed: